crates/metaslang/bindings/src/lib.rs

```diff
@@ -267,7 +271,7 @@ impl<KT: KindTypes + 'static> Bindings<KT> {
 
```

Comment:
@ggiraldez unrelated to the PR, but I wonder if this `TODO` is already fixed?

> // TODO: what should we do if the parent reference
> // cannot be resolved at this point?

Reply:
It's not, but I don't know if there's something meaningful we can do in this case. For now, if we cannot resolve a parent it's simply ignored.

This resolution method is used internally, while resolving other references. Eg. for determining which methods are reachable when resolving a virtual call; or to determine which extension scopes (ie. attached functions) apply in a particular context. Resolving parents in these circumstances should not be a problem unless the user code is invalid, or if there's a bug in the binding rules.

Reply:
> unless the user code is invalid

Which is a possible use case, right?

> if we cannot resolve a parent it's simply ignored

Sounds good. I will use `None` in this case.

Reply:
Yes, it's certainly possible that the user code is invalid and that means we cannot resolve a parent reference. Eg. definining `contract Test is Base {}` without defining `Base`. This failure to resolve is currently ignored. I think my main concern for writing the `TODO` was that while this may impact resolved references, there's no feedback given to the user indicating that this may be a problem. In general, the current behaviour should be ok though.